packages/SwingSet/test/test-xsnap-store.js

```diff
@@ -130,6 +130,14 @@ test('create SES worker, save, restore, resume', async t => {
  * They are also sensitive to the XS code itself.
  */
 test('XS + SES snapshots are long-term deterministic', async t => {
+  /* global globalThis */
```

Comment:
Huh. I did not know we could do this locally. Is it scoped to the block it appears in?

Reply:
Unfortunately, not scoped.

Reply:
It that case, I'd prefer it at the top of the file, since that lack of scoping is surprising and this misunderstanding fails unsafe.

Reply:
Yes let's move to the top